src/gh/components/DF_cloud_mesh_distance/code.py

```diff
@@ -28,7 +28,7 @@ def RunScript(self,
             The cloud-to-mesh component computes the distance between a point cloud and a mesh
 
             :param i_cloud_source: a list of point clouds
-            :param i_beams: a list of DF beams
+            :param i_assembly: an assembly of DF beams
             :param i_signed_flag: calculate the sign of the distances
             :param i_swap: swap source and target
```

Comment:
Just a minor from the PR #60: docstrings here should be removed because we use `metadata.json` for the documentation generation.

Reply:
removed!